api/graphql/application_rounds/application_round_types.py

```diff
@@ -83,7 +83,6 @@ class Input:
     class Meta:
         model = ApplicationRound
         fields = [
-            "id",
```

Comment:
Removing this looks like a breaking change. Hopefully client is not using this field.

Reply:
It shouldn't be in use and I've discussed that these ids are not our way to go, rather the `pk`